### PR TITLE
chore: github workflows

### PR DESCRIPTION
**Pull request by xhyrom**

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    name: Lint
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2
+
+      - name: Install Node
+        uses: actions/setup-node@v2
+        with:
+          node-version: 14
```
Comment (Tolfx) [inline .github/workflows/test.yml]: Use strategy matrix on node versions.

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2
+
+      - name: Install Node
+        uses: actions/setup-node@v2
+        with:
+          node-version: 14
```
Comment (Tolfx) [inline .github/workflows/test.yml]: Use strategy matrix on node versions.

File: .github/workflows/release.yml [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    name: publish
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2
+
+      - name: Install Node
+        uses: actions/setup-node@v2
+        with:
+          node-version: 14
```
Comment (Tolfx) [inline .github/workflows/release.yml:16]: Use strategy matrix on node versions.

File: .github/workflows/test.yml [hunk 4]
```diff
@@ -0,0 +1,42 @@
+name: "Test"
+
+on: [ workflow_dispatch ]
```
Comment (Tolfx) [inline .github/workflows/test.yml]: Check on pull_request and push on dev/master

**Review by Tolfx — DISMISSED** [review]
LGTM!

**Review by Tolfx — DISMISSED** [review]

File: .github/workflows/release.yml (hunk 3 above)
Comment (AngeloCore) [inline .github/workflows/release.yml]: use v17

File: .github/workflows/test.yml [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+      - name: Install Node ${{ matrix.node }}
+        uses: actions/setup-node@v2
+        with:
+          node-version: ${{ matrix.node }}
+          registry-url: https://registry.npmjs.org/
+
+      - name: Install
+        run: npm ci
+
+      - name: build
```
Comment (Tolfx) [inline .github/workflows/test.yml]: Change name to `lint` instead of `build`

**Review by Tolfx — CHANGES_REQUESTED** [review]

**Review by Tolfx — DISMISSED** [review]

**Review by AngeloCore — APPROVED** [review]

**Pull request merged**